easybuild/tools/options.py

```diff
@@ -1091,7 +1135,7 @@ def _postprocess_config(self):
 
             # paths specified to --robot have preference over --robot-paths
             # keep both values in sync if robot is enabled, which implies enabling dependency resolver
-            self.options.robot_paths = [os.path.abspath(path) for path in self.options.robot + self.options.robot_paths]
+            self.options.robot_paths = [os.path.abspath(p) for p in self.options.robot + self.options.robot_paths]
```

Comment:
Hmmm., doesn't this mean that robot_paths get run twice through abspath?? I don't know if that "+" op belongs to the for args or not...

Reply:
`for p in self.options.robot + self.options.robot_paths` is equivalent with `for p in (self.options.robot + self.options.robot_paths)`, is that the question?

`robot_paths` is already covered higher up, sure, but `os.path.abspath` is *cheap* for absolute paths anyway, so that's fine?

Reply:
Yeah, but then wouldn't [os.path.abspath(p) for p in self.options.robot] + self.options.robot_paths be even cheaper?

Reply:
True, I can change it to that (but we're basically talking about skipping of `if p.startswith('/')`)

Reply:
fixed in 94e987603